### DISCUSSION TITLE
Disable Announcement bar "auto hide"

### DISCUSSION THREAD

**Discussion by DininduSenanayake** [squidfunk/mkdocs-material#5142] [Q&A] [answered]
Repository in question : https://genomicsaotearoa.github.io/data-management-resources/

Is there a way to stop the `announcement` bar  being auto-hidden while scrolling down. I would like it be there **all** the time, similar to the header

**Comment by squidfunk** [comment, marked as answer]
Duplicate of #3355. Please try to use the search before asking questions for which solutions already exist. This reduces overhead for you, and for us maintainers, leaving more time to improve this project 😀 Thank you.

**Reply by squidfunk** [reply]
By the way, it took me 1 minute to find this...

**Reply by DininduSenanayake** [reply]
Apologies folks . 100% my fault not being thorough with the reviews of existing threads
